src/services/expenses.service.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const sequelize = require('./../database');
+
+const Expense = sequelize.define('Expense', {
+  id: {
+    type: DataTypes.INTEGER,
+    primaryKey: true,
+  },
+  userId: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  spentAt: {
+    type: DataTypes.DATE,
+    allowNull: false,
+    defaultValue: DataTypes.NOW,
+  },
+  title: {
+    type: DataTypes.TEXT,
+    allowNull: false,
+  },
+  amount: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  category: {
+    type: DataTypes.TEXT,
+    allowNull: false,
+  },
+  note: {
+    type: DataTypes.TEXT,
+    allowNull: false,
+  },
+}, {
+  tableName: 'expenses',
+  updatedAt: false,
+  createdAt: false,
+});
+
+const getAllExpenses = async(userId, categories, from, to) => {
+  let filteredExpenses = await Expense.findAll();
+
+  if (userId) {
+    filteredExpenses
+      = filteredExpenses.filter((expense) => expense.userId === +userId);
+  }
+
+  if (categories) {
+    filteredExpenses = filteredExpenses.filter((expense) =>
+      categories.includes(expense.category)
+    );
+  }
+
+  if (from && to) {
+    filteredExpenses = filteredExpenses.filter(
+      (expense) =>
```

Comment:
Why do you query all expenses and then filter them instead of querying only what you actually need from the database?